starters/deno-oak-denodb/src/rest/services/healtcheck_service.ts

```diff
@@ -17,7 +17,7 @@ export async function isCacheRunning(): Promise<boolean> {
 				}, 2000);
 			}),
 		]);
-		isRunning = (await pingResponses)[0] === 'PONG';
+		isRunning = (await pingResponses) === 'PONG';
```

Comment:
`Promise.race()` takes an array of promises but returns a sing promise (the fastest to resolve)

Reply:
Yes, exactly. Nice fix :)